.github/workflows/common-flow-host.yml

```diff
@@ -0,0 +1,83 @@
+name: default (common) flow for host runner
+
+on:
+  workflow_call:
+    inputs:
+      env:
+        required: true
+        type: string
+      san:
+        type: string
+      get-redis:
+        type: boolean
+
+env:
+  MODE: "sudo"
+
+jobs:
+  common-flow:
+    runs-on: ${{ inputs.env }}
+# From this point on, this file should be identical to the other common flow.
+    defaults:
+      run:
+        shell: bash -l -eo pipefail {0}
+    steps:
+      - name: Deps checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
```

Comment:
```suggestion
        uses: actions/checkout@v3
        with:
            submodules: recursive
            set-safe-directory: "'*'"
```

Reply:
setup issues. changing this in the next PR